mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -1795,18 +1793,11 @@ void CMomentumPlayer::EnablePracticeMode()
     if (m_bHasPracticeMode)
         return;
 
+    if (g_pRunSafeguards->IsSafeguarded(RUN_SAFEGUARD_PRACTICEMODE))
+        return;
+
     if (g_pMomentumTimer->IsRunning())
     {
-        if (mom_practice_safeguard.GetBool())
-        {
-            const auto safeGuard = (m_nButtons & (IN_FORWARD | IN_MOVELEFT | IN_MOVERIGHT | IN_BACK | IN_JUMP | IN_DUCK | IN_WALK)) != 0;
-            if (safeGuard)
-            {
-                Warning("You cannot enable practice mode while moving when the timer is running! Toggle this with \"mom_practice_safeguard\"!\n");
-                return;
-            }
-        }
-
         if (mom_practice_warning_enable.GetBool())
```

Comment:
Simplify this check by combining the previous if statement with this one

Reply:
Hey, this wasn't resolved. What I meant was to do:

```
if (g_pMomentumTimer->IsRunning() && mom_practice_warning_enable.GetBool())
{
    UTIL_ShowMessage( . . . );
    . . . 
}
```

Reply:
:P

Reply:
oh woops missed this 1